File: ChatService/Models/Message.cs
using MongoDB.Bson.Serialization.Attributes;

namespace ChatService.Models
{
    [BsonNoId]
[BsonIgnoreExtraElements]
    public record Message
    {
        public String MessId { get; init; }
        public String User { get; init; }
        public String Text { get; init; }
        // [BsonId]
        public string mid { get; init; }
        public UInt64 Timestamp { get; init; }
    }
}

File: frontend/utils/Message.js
//klasa wiadomosci i jej atrybuty
class Message {
    constructor(messid, mid, user, text) {
        this.messid = messid;
        this.mid = mid;
        this.user = user;
        this.text = text;
        this.timestamp = Math.floor(new Date().getTime()/1000.0)
    }
}
export default Message;